packages/ffe-datepicker-react/src/index.d.ts

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+interface IinputProps {
+    className?: string;
+    id?: string;
+}
+
+export interface DatepickerProps
+    extends React.SelectHTMLAttributes<HTMLSelectElement> {
+    'aria-invalid'?: string;
+    ariaInvalid?: boolean | string;
+    calendarAbove?: boolean;
+    hideErrors?: boolean;
+    onValidationComplete?: Function;
+    inputProps?: IinputProps;
+    label?: string;
+    language?: string;
+    maxDate?: string;
+    minDate?: string;
+    onChange: Function;
```

Comment:
Her ville jeg brukt React.ChangeEventHandler i stedet for Function

Reply:
Da er det fikset. Men hva er egentlig grunnen til at `React.ChangeEventHandler` er bedre egnet som type til onChange her? Er det fordi `Function` blir for vagt eller generelt?

Reply:
Ja, React.ChangeEventHandler vil være mer spesifikt (siden det er det formet bruker). Men, Function vil fungere det også, så det var ikke helt galt slik det stod heller.

Reply:
Åja, det gir mening. Takk for feedback.